lua/ulx/modules/sh/cfc_arrest.lua

```diff
@@ -3,14 +3,24 @@ local cmd = CFCUlxCommands.arrest
 
 CATEGORY_NAME = "Cleanup"
 
+local function isTargetPlayer( ply, targetPlayers )
+    for _, p in pairs( targetPlayers ) do
+        if p == ply then
+            return true
+        end
+    end
+
+    return false
+end
+
 function cmd.arrest( callingPlayer, targetPlayers )
     local entities = ents.GetAll()
     local entCount = 0
     local entCounts = {}
 
     for _, ent in ipairs( entities ) do
         local owner = ent.CPPIGetOwner and ent:CPPIGetOwner()
-        if owner and entCounts[owner] then
+        if owner and isTargetPlayer( owner ) then
```

Comment:
Instead we could make a lookup table first, which would only loop through the players once

```lua
function cmd.arrest( callingPlayer, targetPlayers )
    local entities = ents.GetAll()
    local entCount = 0
    local entCounts = {}

    local targetLookup= {}
    for _, ply in ipairs( targetPlayers ) do
        targetLookup[ply] = true
    end

    for _, ent in ipairs( entities ) do
        local owner = ent.CPPIGetOwner and ent:CPPIGetOwner() or ent:GetOwner()
        if owner and targetLookup[owner] then
```